Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxDefaultModeBus.h

```diff
@@ -60,6 +60,10 @@ namespace WhiteBox
         //! Returns the hovered polygon handle.
         //! @note If no polygon is currently being hovered then an empty PolygonHandle is returned.
         virtual Api::PolygonHandle HoveredPolygonHandle() const = 0;
+        //! Hide the selected Edge.
+        virtual void HideSelectedEdge() const = 0;
+        //! Hide the selected Vertex.
+        virtual void HideSelectedVertex() const = 0;
```

Comment:
update: Should these be `const`? It seems to me like this is a mutable operation, they probably shouldn't be `const` 🤔

Reply:
You are correct, fixing.

Reply:
Nice one thanks!